makeRelease.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 if [ "$#" -ne 2 ]; then
-    echo "Usage:\n  ./makeRelease 1.2.3rc0 Directory/"
+    printf "Usage:\n  ./makeRelease 1.2.3rc0 Directory/"
```

Comment:
Printf does not add a newline automatically. Please add one to the end of the line if you want to use printf.